protocols/netty/src/main/java/org/apache/james/protocols/netty/AbstractAsyncServer.java

```diff
@@ -93,7 +93,7 @@ public synchronized void bind() throws Exception {
         ServerBootstrap bootstrap = new ServerBootstrap();
         bootstrap.channel(NioServerSocketChannel.class);
 
-        bossGroup = new NioEventLoopGroup(NamedThreadFactory.withName(jmxName + "-boss"));
+        bossGroup = new NioEventLoopGroup(2, NamedThreadFactory.withName(jmxName + "-boss"));
```

Comment:
FYI These types of hard coded thread pool sizes are a big part of what this pull request is trying to fix.